test/integration_test.py

```diff
@@ -714,3 +724,52 @@ def test_mini_dss_failures(self):
                 mini_dss.get_file(uuid, version, 'aws')
             with self.assertRaises(self.SpecialError):
                 mini_dss.get_native_file_url(uuid, version, 'aws')
+
+
+class AzulChaliceLocalIntegrationTest(AzulTestCase):
+    host = '127.0.0.1'
+    port = 8000
+    url = furl(scheme='http', host=host, port=port)
+    local_chalice_server = None
+    server_thread = None
+
+    @classmethod
+    def setUpClass(cls, config=config) -> None:
+        super().setUpClass()
+        service_app = Path(config.project_root) / 'lambdas' / 'service'
+        localfactory = factory.CLIFactory(str(service_app))
```

Comment:
```suggestion
        factory = factory.CLIFactory(str(service_app))
```

Reply:
This rename was giving me an invalid reference to `factory.CLIFactory()` factory. So I opted for `factory_` instead.